tool/site/client/src/pages/sendPreSignPage.tsx

```diff
@@ -88,13 +134,33 @@ async function retrievePrescription(baseUrl: string, prescriptionId: string): Pr
   return getResponseDataIfValid(response, isBundle)
 }
 
-async function sendSignRequest(baseUrl: string) {
+async function sendSignRequest(baseUrl: string, sendPageFormValues: SendPreSignPageFormValues) {
+  await updateEditedPrescriptions(sendPageFormValues, baseUrl)
   const response = await axiosInstance.post<SignResponse>(`${baseUrl}prescribe/sign`)
   const signResponse = getResponseDataIfValid(response, isSignResponse)
   redirect(signResponse.redirectUri)
   return signResponse
 }
 
+async function updateEditedPrescriptions(sendPageFormValues: SendPreSignPageFormValues, baseUrl: string) {
+  if (sendPageFormValues.nominatedOds) {
+    const prescriptions = (await axiosInstance.get(`${baseUrl}prescriptions`)).data as Array<Bundle>
+    prescriptions.forEach(prescription => {
+      const medicationRequests = getMedicationRequestResources(prescription)
+      medicationRequests.forEach(medication => medication.dispenseRequest.performer.identifier.value = sendPageFormValues.nominatedOds)
+    })
+    const newPrescriptions: Array<Bundle> = prescriptions
+      .map(p => Array(parseInt(sendPageFormValues.numberOfCopies)).fill(p).map(p => clone(p))
+      ).flat()
+    newPrescriptions.forEach(p => updateBundleIds(p))
+    await axiosInstance.post(`${baseUrl}prescribe/edit`, newPrescriptions)
+  }
+}
+
+function clone(p: any): any {
```

Comment:
could be worth adding `lodash` into the project, has some nice utils that we could use across the project